src/main/java/cholog/wiseshop/api/product/controller/ProductController.java

```diff
@@ -37,11 +38,11 @@ public ResponseEntity<Void> modifyProduct(@PathVariable Long id,
     }
 
     @PatchMapping("/products/{id}/price")
-    public ResponseEntity<Void> modifyProductPrice(
+    public ResponseEntity<Void> modifyProductPriceAndStock(
         @PathVariable Long id,
-        @RequestBody ModifyProductPriceRequest request
+        @RequestBody ModifyProductPriceAntStockRequest request
```

Comment:
```suggestion
        @RequestBody ModifyProductPriceAndStockRequest request
```
오타난거 같은데 Ant -> And 맞나요?

Reply:
넹 ㅎ 위에서도 오타 말해줘서 고쳐두었습니당...^..^^